x-pack/plugins/osquery/public/saved_queries/saved_query_flyout.tsx

```diff
@@ -54,7 +59,7 @@ const SavedQueryFlyoutComponent: React.FC<AddQueryFlyoutProps> = ({ defaultValue
         onClose={onClose}
         aria-labelledby="flyoutTitle"
         // eslint-disable-next-line react-perf/jsx-no-new-object-as-prop
-        maskProps={{ style: 'z-index: 6000' }} // For an edge case to display above the alerts flyout
+        maskProps={isExternal && { style: 'z-index: 6000' }} // For an edge case to display above the alerts flyout
```

Comment:
💅  Instead of adding an eslint exception, you could also extract `{ style: 'z-index: 6000' }` to a variable outside the scope of this component.

Reply:
Sure, will try! :)